src/Bicep.Core/Workspaces/SourceFileGrouping.cs

```diff
@@ -22,21 +22,21 @@ public record UriResolutionError(
     DiagnosticBuilder.ErrorBuilderDelegate ErrorBuilder,
     bool RequiresRestore);
 
-public class SourceFileGrouping : ISourceFileLookup
+public class SourceFileGrouping : IArtifactFileLookup
```

Comment:
I admit I don't understand the reason for this change and the uri ->fileUri change.

Reply:
Since a provider artifact file reference is not technically a source file, the name is misleading.